polaris/evaluate/_metric.py

```diff
@@ -75,7 +99,9 @@ class Metric(Enum):
     pr_auc = MetricInfo(fn=average_precision_score, direction="max")
     mcc = MetricInfo(fn=matthews_corrcoef, direction="max")
     cohen_kappa = MetricInfo(fn=cohen_kappa_score, direction="max")
+
     # TODO: adding metrics for multiclass tasks
+    ba = MetricInfo(fn=balanced_accuracy_score, direction="max")
```

Comment:
For clarity, I would use:
```suggestion
    balanced_accuracy = MetricInfo(fn=balanced_accuracy_score, direction="max")
```

Reply:
Also, by placing this under the `TODO` comment, it seems to me this is a multi-task metric? Is that the case? If so, the `MetricInfo.is_multitask` flag should be set and I would update the comment. If not, I would move it above the comment.

Reply:
It's multiclass not multitask.